pkg/controller/namespace/namespace_controller.go

```diff
@@ -79,11 +71,7 @@ func NewNamespaceController(
 
 	// create the controller so we can inject the enqueue function
 	namespaceController := &NamespaceController{
-		kubeClient:                 kubeClient,
-		clientPool:                 clientPool,
```

Comment:
> Were these just not refactored cleanly by previous output?

Looks like at some point the factorization of the controller changed and all the logic moved out into a different type which does the same thing.  I noticed it as I was checking the wiring